configure

```diff
@@ -920,23 +921,39 @@ sub path_parent_dir {
 ## Security
 
 if ($opts{'security'}) {
-  if (! exists $opts{'ssl'}) {
+  unless (exists $opts{'ssl'}) {
     $opts{'ssl'} = '';
     print "Forcing --ssl (security dependency)\n" if $opts{'verbose'};
   }
-  if (! exists $opts{'xerces3'}) {
+  unless (exists $opts{'xerces3'}) {
     $opts{'xerces3'} = '';
     print "Forcing --xerces3 (security dependency)\n" if $opts{'verbose'};
   }
   if ($tests) {
-    # CMake is required in order to compile the Google Test framework
-    # (which security tests rely on).
-    $opts{'cmake'} = which('cmake');
+    # CMake is required in order to compile the Google Test Framework.
+    unless ($opts{'cmake'}) {
+      print "--cmake not specified by user; searching path...\n" if $opts{'verbose'};
+      $opts{'cmake'} = which('cmake');
 
-    die "ERROR: CMake not found on PATH"
-      unless defined $opts{'cmake'};
+      unless ($opts{'cmake'}) {
+        print "CMake not found in path; searching default location...\n"
+          if $opts{'verbose'};
+
+        my $cmake_default =
+          (is_windows() ?
+            (-f 'c:\\Program Files\\CMake\\bin\\cmake.exe' ?
```

Comment:
There is a windows env for the program files path

Reply:
Yes, we need to use the environment for this:
```
ProgramFiles=C:\Program Files
ProgramFiles(x86)=C:\Program Files (x86)
```

Reply:
Alas! Thank you. I shall use that instead :)